packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -275,23 +294,64 @@ function FontCollection( { slug } ) {
 			) }
 
 			{ ! renderConfirmDialog && ! selectedFont && (
-				<FontsGrid>
-					{ fonts.map( ( font ) => (
-						<FontCard
-							key={ font.font_family_settings.slug }
-							font={ font.font_family_settings }
-							onClick={ () => {
-								setSelectedFont( font.font_family_settings );
-							} }
-						/>
-					) ) }
-				</FontsGrid>
+				<VStack spacing={ 0 }>
+					<div className="font-library-modal__fonts-grid__main">
+						{ items.map( ( font ) => (
+							<FontCard
+								key={ font.font_family_settings.slug }
+								font={ font.font_family_settings }
+								onClick={ () => {
+									setSelectedFont(
+										font.font_family_settings
+									);
+								} }
+							/>
+						) ) }
+					</div>
+				</VStack>
 			) }
 		</TabPanelLayout>
 	);
 }
 
-function Footer( { handleInstall, isDisabled } ) {
+function PaginationFooter( { page, totalPages, setPage } ) {
+	return (
+		<Flex justify="center">
+			<Button onClick={ () => setPage( 1 ) } disabled={ page === 1 }>
+				&lt;&lt;
```

Comment:
The Revisions and block patterns pagination uses these type of symbols: « ‹
See https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-patterns-paging/index.js#L43

These buttons must have an aria label, the symbols alone can not be used as the accessible name.
They should be focusable also when they are disabled, so that the focus order remains the same.

Reply:
Thank you.  Changed icons, added aria labels and corrected the focus-ability in [this commit](https://github.com/WordPress/gutenberg/pull/58794/commits/406518ccd7c2c262dcd401c4f08ae73ec197c889).